catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -171,6 +171,35 @@
             type="Integer"
             default="25"
             required="true"/>
+
+        <AD id="queryFeedbackEnabled"
```

Comment:
You might want to talk to @ricklarsen  about updating/improving the documentation for these new metatype fields.